minishift/check_hosts.sh

```diff
@@ -0,0 +1,13 @@
+IP=$(minishift ip)
```

Comment:
should we do a `set -e` here?

Reply:
yes added https://github.com/fabric8-services/fabric8-wit/pull/1781/commits/289068302bccce44f67586c43bfc7e507410bf69